boltdesignsystem/bolt
# [DO NOT MERGE] Feature/native button poc

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-121](https://pegadigitalit.atlassian.net/browse/DS-121)

## Summary

Allowing a native (non-web-component) version of the button.

## Details

Following the example set in native-link-poc to create a version of the button that is not a web component in an effort to make progress toward our new component organization scheme.

## How to test

Make sure that button.twig is showing the non-web-component version (currently _button-replacement.twig) by setting noWC to true, then confirm that it is working as intended.
